packages/site-components/src/NavigationItem/NavigationItem.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { AriaAttributes, ForwardedRef, MouseEvent, ReactElement } from 'react';
+import { ComponentPropsWithoutRef, forwardRef, MouseEventHandler } from 'react';
+import { makePrefixer } from '@salt-ds/core';
+import { clsx } from 'clsx';
+import { ExpansionIcon } from './ExpansionIcon';
+
+import { useWindow } from '@salt-ds/window';
+import { useComponentCssInjection } from '@salt-ds/styles';
+
+import navigationItemCss from '@salt-ds/core/dist-es/navigation-item/NavigationItem.css.js';
+
+type OptionalPartial<T, K extends keyof T> = Partial<Pick<T, K>>;
+
+export interface NavigationItemElementProps<T extends HTMLAnchorElement | HTMLButtonElement>
+  extends Pick<AriaAttributes, 'aria-label' | 'aria-expanded' | 'aria-current'> {
+  /**
+   * Item's children.
+   */
+  children: ReactElement;
+  /**
+   * Item's className.
+   */
+  className: string;
```

Comment:
`?`?

Reply:
We always pass a className, it's not optional, unlike onClick, which we only pass for parent elements (groups)